FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/FleetProvisioningDemoExample.c

```diff
@@ -124,29 +124,30 @@
  * @brief Time in seconds to wait between retries of the demo loop if
  * demo loop fails.
  */
-#define fpdemoDELAY_BETWEEN_DEMO_RETRY_ITERATIONS_SECONDS    ( 5 )
+#define fpdemoDELAY_BETWEEN_DEMO_RETRY_ITERATIONS_SECONDS    ( 10 )
 
 /**
  * @brief Size of buffer in which to hold the certificate signing request (CSR).
  */
-#define fpdemoCSR_BUFFER_LENGTH                              2048
+#define fpdemoCSR_BUFFER_LENGTH                              4096
 
 /**
  * @brief Size of buffer in which to hold the certificate.
  */
-#define fpdemoCERT_BUFFER_LENGTH                             2048
+#define fpdemoCERT_BUFFER_LENGTH                             4096
 
 /**
  * @brief Size of buffer in which to hold the certificate id.
  *
- * See https://docs.aws.amazon.com/iot/latest/apireference/API_Certificate.html#iot-Type-Certificate-certificateId
+ * @note Has a maximum length of 64 for more information see the following link
+ * https://docs.aws.amazon.com/iot/latest/apireference/API_Certificate.html#iot-Type-Certificate-certificateId
  */
 #define fpdemoCERT_ID_BUFFER_LENGTH                          64
 
 /**
  * @brief Size of buffer in which to hold the certificate ownership token.
  */
-#define fpdemoOWNERSHIP_TOKEN_BUFFER_LENGTH                  512
+#define fpdemoOWNERSHIP_TOKEN_BUFFER_LENGTH                  1024
```

Comment:
Use of an ECDSA key can require a larger buffer to hold some of this info, double the size of these buffers to account for this change